Examples/Modules/laser_injection_from_file/analysis3d.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
```

Comment:
Just asking: are `analysis2d.py` and `analysis3d.py` quite different? If they are basically doing the same thing, could we merge them together to reduce duplication?

Reply:
I think that the two scripts could be merged. They are indeed very similar...

Reply:
I have merged them into a single `analysis.py` script. The choice of the analysis which should be performed is made with a command line argument.